js/src/block-editor/components/edit.js

```diff
@@ -135,6 +125,10 @@ const Edit = ( { block, blockProps } ) => {
 									attributes={ blockProps.attributes }
 									className="genesis-custom-blocks-editor__ssr"
 									httpMethod="POST"
+									urlQueryArgs={ { inner_blocks: innerBlocks
```

Comment:
`<ServerSideRender />` has no native way to pass it the InnerBlocks HTML, so we use the `urlQueryArgs` prop.

Then, we look for that query arg in `InnerBlocks::validate()` with `filter_input()`.